devtools/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM continuumio/miniconda3
+
+SHELL ["/bin/bash", "-c"]
+
+COPY . fah-xchem
+
+RUN conda env create -n fah-xchem -f fah-xchem/environment.yml \
+ && source activate fah-xchem \
```

Comment:
Should there be a `conda clean` in here to clean up all the staged packages?

Reply:
Yes, good catch. Adding that now. We certainly don't need cached, compressed versions of the packages lying about.